components/04-views/catalog/_catalog.scss

```diff
@@ -1,37 +1,55 @@
+//product-grid
+$product-grid-padding             : 10px !default;
+$product-grid-margin              : 15px !default;
+$product-grid-breakpoint-medium   : $screen-m !default;
+$product-grid-breakpoint-large    : $screen-l !default;
+
+//product-list
+$product-list-padding             : 10px !default;
+$product-list-margin              : 10px !default;
+$product-list-border-color        : $gray-light !default;
+$product-list-border              : 1px solid $product-list-border-color !default;
+$product-list-name-font-size      : $font-size-small !default;
+$product-list-details-color       : $gray-lighter !default;
+$product-list-details-font-size   : $font-size-extra-small !default;
+$product-list-img-box-padding     : 0 $product-grid-padding 0 0 !default;
+$product-list-img-box-border-color: $gray-light !default;
+$product-list-img-box-border      : 1px solid $product-list-img-box-border-color !default;
+
 .products-grid {
     display: flex;
     flex-flow: row wrap;
+    justify-content: flex-start;
     width: 100%;
     margin: 0;
     padding: 0;
-    justify-content: flex-start;
     list-style-type: none;
 
-    @include mq($screen-m) {
-        padding: 10px 0 10px 20px;
+    @include mq($product-grid-breakpoint-medium) {
+        padding: $product-grid-padding 0 $product-grid-padding ($product-grid-padding * 2);
```

Comment:
Are you sure that in every case we would like to have a left padding that is two times wider than base padding and no left padding at all.

Reply:
i changed here some styles b/c it was weird